content/docs/command-reference/import.md

```diff
@@ -39,12 +39,12 @@ the data source. Both HTTP and SSH protocols are supported for online repos
 to an "offline" repo (if it's a DVC repo without a default remote, instead of
 downloading, DVC will try to copy the target data from its <abbr>cache</abbr>).
 
-The `path` argument is used to specify the location of the target to be
-downloaded within the source repository at `url`, including those nested inside
-directories [tracked as a whole](/doc/command-reference/add#example-directory).
-`path` can specify any file or directory in the source repo, either tracked by
-DVC, or by Git. Note that DVC-tracked targets should be found in a `dvc.yaml` or
-`.dvc` file of the project.
+The `path` argument is used to specify the location of the target to download
+within the source repository at `url`, including those nested inside
```

Comment:
same here - feels like including should go later

How about this version:

`It can be any path to any Git or DVC-tracked directory or file in the project.`

to replace effectively everything after "at `url`," ?

Reply:
Yes, already updated all these.

Your suggestion sounds good and simple but we need to specify that paths inside tracked directories are supported.